configuration/scripts/machines/env.gadi_intel

```diff
@@ -0,0 +1,56 @@
+#!/bin/csh -f
+
+set inp = "undefined"
+if ($#argv == 1) then
+  set inp = $1
+endif
+
+if ("$inp" != "-nomodules") then
+
+  source /etc/profile.d/modules.csh
+
+  module load intel-compiler
+  module load openmpi
+  
+  if ($?ICE_IOTYPE) then
+    if ($ICE_IOTYPE =~ pio*) then
+      if ($ICE_IOTYPE == "pio1") then
+        # we don't have pio1 installed anywhere  
+        module load pnetcdf
+        module load netcdf
+        module load pio
+      else
+        module load parallelio
+      endif
+    else 
+      module load netcdf
+    endif
+  endif
+
+  if ($?ICE_BFBTYPE) then
+    if ($ICE_BFBTYPE =~ qcchk*) then
+      # conda/analysis has the required librarys, skip building from cice yaml file
+      module use /g/data/hh5/public/modules
+      module load conda/analysis
+      # conda env create -f ../../configuration/scripts/tests/qctest.yml
+      # conda activate qctest
+    endif
+  endif
+
+endif
+
+setenv ICE_MACHINE_MACHNAME gadi
+setenv ICE_MACHINE_MACHINFO "Intel Xeon Scalable"
+setenv ICE_MACHINE_ENVNAME intel
+# setenv ICE_MACHINE_ENVINFO ${module list}
```

Comment:
I think you might also have an error here.  It's possible the failure you were getting was unrelated to the overall problem.  If you continue to have problems, my guess is that it's because these env variables probably have to be defined.  Try changing to

setenv ICE_MACHINE_ENVINFO unknown

(or similar) and see if that fixes the problem.